presto-spi/src/main/java/com/facebook/presto/spi/connector/ConnectorContext.java

```diff
@@ -39,4 +43,9 @@ default PageIndexerFactory getPageIndexerFactory()
     {
         throw new UnsupportedOperationException();
     }
+
+    default MBeanServer getMBeanServer()
+    {
+       return getPlatformMBeanServer();
```

Comment:
Make this `throw new UnsupportedOperationException()` rather than returning the wrong one